APITests/APITests.swift

```diff
@@ -0,0 +1,65 @@
+@testable import WordPressKit
+import OHHTTPStubs
+import XCTest
+
+class APITests: XCTestCase {
+
+    func testExample() throws {
+        let remote = PluginServiceRemote(wordPressComRestApi: WordPressComRestApi.defaultApi())
+
+        stub(
+            condition: { _ in return true },
+            response: { _ -> HTTPStubsResponse in
+                return HTTPStubsResponse(
+                    data: responseWithDouble.data(using: .utf8)!,
+                    statusCode: 200,
+                    headers: .none
+                )
+            }
+        )
```

Comment:
Because the API is now [fixed](https://github.com/WordPress/wordpress.org/commit/1ed8c6633ed3b6aec56d246c209320c3fcc12a53) I had to hijack the response in order to verify the test would have failed.